numpy/core/src/npysort/x86-qsort.dispatch.cpp

```diff
@@ -808,7 +812,7 @@ NPY_CPU_DISPATCH_CURFX(x86_quicksort_int)(void *arr, npy_intp arrsize)
 {
     if (arrsize > 1) {
         qsort_<vector<npy_int>, npy_int>((npy_int *)arr, 0, arrsize - 1,
-                                         2 * log2(arrsize));
+                                         2 * (npy_int)log2(arrsize));
```

Comment:
Hmm. I don't know if this even matters much, but could do  `(npy_int)(2 * log2(arrsize)));`